package.json

```diff
@@ -66,6 +70,26 @@
     "browser": {
       "./lib/platform/node.js": "./lib/platform/browser.js",
       "./lib/platform/node.mjs": "./lib/platform/browser.mjs"
+    },
+    "exports": {
+      ".": {
+        "types": "./lib/advanced/index.d.ts",
+        "require": "./lib/advanced/index.js",
+        "import": "./lib/advanced/index.mjs"
+      },
+      "./platform": {
+        "node": {
+          "types": "./lib/platform/node.d.ts",
+          "require": "./lib/platform/node.js",
+          "import": "./lib/platform/node.mjs"
+        },
+        "browser": {
+          "types": "./lib/platform/browser.d.ts",
+          "require": "./lib/platform/browser.js",
+          "import": "./lib/platform/browser.mjs"
+        }
+      },
```

Comment:
This shouldn't be necessary, it's not supposed to be exposed to consumers.

Reply:
The browser field exists for some legacy bundlers and is not supported by the whole ecosystem widely. eg. TypeScript